src/js/digit.js

```diff
@@ -0,0 +1,21 @@
+export default class Digit {
+ #totalNumber;
+ constructor(stringNumber = '') {
+  const digitNumber = stringNumber.split('.');
+  this.#totalNumber = digitNumber[0] || '';
+ }
+ set appendNumber(stringNumber) {
+  const numberUnitLimit = 3;
+  if (this.#totalNumber.length >= numberUnitLimit) return;
+  this.#totalNumber += stringNumber;
+ }
+
+ get getNumber() {
+  if (this.#totalNumber === '') return 0;
+  return Number(this.#totalNumber);
+ }
```

Comment:
setter보다 메서드일때 appendNumber이 적절해보여요
getNumber도요!
```suggestion
 set totalNumber(stringNumber) {
  const numberUnitLimit = 3;   // constants로 분리하는것도 좋아보여요
  if (this.#totalNumber.length >= numberUnitLimit) return;
  this.#totalNumber += stringNumber;
 }

 get totalNumber() {
  if (this.#totalNumber === '') return 0;
  return Number(this.#totalNumber);
 }
```